src/projectM-pulseaudio/qprojectM-pulseaudio.cpp

```diff
@@ -173,30 +173,30 @@ std::string read_config()
 	strcpy ( projectM_config+strlen ( PROJECTM_PREFIX ), CONFIG_FILE );
 	projectM_config[strlen ( PROJECTM_PREFIX ) +strlen ( CONFIG_FILE ) ]='\0';
 	printf ( "dir:%s \n",projectM_config );
-	home=getenv ( "HOME" );
+	home=getenv ( "XDG_CONFIG_HOME" );
```

Comment:
is this guaranteed to be set? about on say, macOS?

Reply:
As of macOS version 10.14.4, `$XDG_CONFIG_HOME` is not set.

We can do a check to see if the variable is set, and in the case that it isn't use a sane default. (Maybe continue to use `$HOME` and leave the current behavior as is.)